src/components/HomePage/index.tsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import CategoryCards from 'components/CategoryCards';
+import { useGetProductsWithImagesQuery } from 'redux/productsApi';
 import Header from 'components/Header';
+import HomePageModal from 'components/HomePageModal';
 import styles from './index.module.scss';
-import { useGetProductsWithImagesQuery } from 'redux/productsApi';
 
 const HomePage = () => {
   const { data } = useGetProductsWithImagesQuery({ page: 0, size: 9 });
   console.log(data);
```

Comment:
Please check, I'm not sure that we need this data in the console.

Reply:
data is not used, so eslint shows an error without a console

Reply:
so, this hook is not needed here, pls remove it too